wpilibcExamples/src/main/cpp/examples/StateSpaceDifferentialDriveSimulation/cpp/RobotContainer.cpp

```diff
@@ -26,6 +26,11 @@ RobotContainer::RobotContainer() {
   ConfigureButtonBindings();
 
   // Set up default drive command
+  // A split-stick arcade command, with forward/backward controlled by the left
+  // hand, and turning controlled by the right.
+  // If you are using the keyboard as a joystick, it is recommended that you go 
+  // to the following link to edit the settings of the keyboard.
```

Comment:
Maybe we can slightly reword this to be "... to read about editing the keyboard settings"